packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1582,7 +1582,8 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     super.initState();
     _clipboardStatus?.addListener(_onChangedClipboardStatus);
     widget.controller.addListener(_didChangeTextEditingValue);
-    _focusAttachment = widget.focusNode.attach(context);
+    if (!widget.focusNode.hasAttachment)
```

Comment:
I think it will be cleaner if we document the focusnode need to be attached before passing into the EditableText, but I think that will be a pretty bad breaking change. so I think this looks fine as is.

Reply:
Is EditableText widely used by itself? If so i don't see another way unfortunately. if there is no wrapper widget the editable text would need to attach.